pipeline.py

```diff
@@ -154,7 +154,8 @@ def pipeline(config, dirpath, device, flow_logger, event_logger):
             if config['analysis'].get('gene_analysis'):
                 analyser.gene_analysis()
 
-        analyser.perform_downstream_anlaysis()
+        analyser.full_samples_downstream_anlaysis()
+        analyser.test_samples_downstream_anlaysis()
```

Comment:
Does this mean, the downstream analysis will run for both full data and test data when ran using pipeline run?

Reply:
Yes, but that depends upon the config. The config now has 2 sections, `full` and `test`, so whichever analysis is wherver we will be running them there. Ideally and hopefully people won't be putting the same analysis in both parts (unless they are testing something out)